app/src/main/java/com/heapixLearn/discovery/domain/contact/DumbUIContactManager.java

```diff
@@ -0,0 +1,22 @@
+package com.heapixLearn.discovery.domain.contact;
+
+import com.heapixLearn.discovery.logic.contact.LogicContact;
+import com.heapixLearn.discovery.logic.contact.contacts_domain_interfaces.UIContactManager;
+
+public class DumbUIContactManager implements UIContactManager {
```

Comment:
@dianaprimshits unused class, should be removed

Reply:
@YodaMike Fixed